static/api/extensions/redhat/bootc/1.14.0/README.md

```
@@ -0,0 +1,351 @@
+
+1. Go to **Extensions** in the navbar.
+2. Click on the **Catalog** tab.
+3. Install the extension.
+
+![](https://raw.githubusercontent.com/podman-desktop/extension-bootc/main/docs/img/catalog_install.gif)
+
+### Nightly version
+
+A version of the extension using the latest commit changes can be installed via the **Install custom...** button with the following link:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)